pyproject.toml

```diff
@@ -13,8 +13,108 @@
 # limitations under the License.
 
 [build-system]
-build-backend = "setuptools.build_meta:__legacy__"
+build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools",
+    "setuptools>=64.0.0",
     "cython>=0.29,<0.30",
+    "tomli  ; python_version < '3.11'",
 ]
+
+[project]
+name = "ucx-py"
+dynamic = [
+    "version",
+]
+description = "Python Bindings for the Unified Communication X library (UCX)"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "BSD-3-Clause" }
+requires-python = ">=3.6"
+dependencies = [
+    "numpy",
+    "psutil",
```

Comment:
```suggestion
```

Reply:
While we're at it, I think we can remove `psutil` that isn't anymore used by UCX-Py. Alternatively, I can open a separate PR to remove other references as well.

Reply:
Would suggest we handle that in a separate PR. There's already a few moving parts here

Reply:
Agreed, I opened https://github.com/rapidsai/ucx-py/pull/896 to address that.

Reply:
@jakirkham https://github.com/rapidsai/ucx-py/pull/896 is now merged, could you please make sure this to apply this suggestion to the PR?

Reply:
Yep merged that change in and dropped `psutil`. Thanks Peter! 🙏